src/modules/pageIcons/pageIcons.ts

```diff
@@ -136,15 +136,22 @@ export const processLinkItem = async (linkItem: HTMLElement) => {
 export const setStyleToLinkItem = async (linkItem: HTMLElement, pageProps: propsObject) => {
     linkItem.classList.remove('awLi-stroke');
     // icon
-    const pageIcon = pageProps['icon'];
+    const pageIcon = pageProps['icon-url'] ? `<img src="${pageProps['icon-url']}">` : pageProps['icon'];
```

Comment:
there is solution for local files - https://github.com/yoyurec/logseq-banners-plugin/blob/main/src/banners.ts#L549-L552

Reply:
Cool that should work for image in asset. I will try it